tests/plugins/balloontoolbar/positioning.js

```diff
@@ -160,6 +161,28 @@
 
 				wait();
 			} );
+		},
+
+		// #1496
+		'test panel reposition': function( editor, bot ) {
+			bot.setData( '<img src="' + bender.basePath + '/_assets/lena.jpg">', function() {
+				var balloonToolbar = new CKEDITOR.ui.balloonToolbarView( editor, {
+						width: 100,
+						height: 200
+					} ),
+					markerElement = editor.editable().findOne( 'img' ),
+					options = { foo: 'bar' },
+					spy;
+
+				balloonToolbar.attach( markerElement );
+
+				spy = sinon.spy( balloonToolbar, 'attach' );
+
+				balloonToolbar.reposition( options );
+
+				assert.areSame( markerElement, spy.args[ 0 ][ 0 ] );
```

Comment:
Probably the element is the same, yet tests fail due to the fact that element's wrappers are different. [`equals`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_dom_element.html#method-equals) should be used here.